docs/docusaurus/docs/components/examples_under_test.py

```diff
@@ -8,6 +8,36 @@
 
 docs_tests = []
 
+docs_examples_customize_expectations = [
+    IntegrationTestFixture(
+        # To test, run:
+        # pytest --docs-tests -k "docs_example_define_a_custom_expectation_class" tests/integration/test_script_runner.py
+        name="docs_example_define_a_custom_expectation_class",
+        user_flow_script="docs/docusaurus/docs/core/customize_expectations/_examples/define_a_custom_expectation_class.py",
+        data_dir="docs/docusaurus/docs/components/_testing/test_data_sets/single_test_file",
+        # data_context_dir="",
+        backend_dependencies=[],
+    ),
+    IntegrationTestFixture(
+        # To test, run:
+        # pytest --docs-tests -k "doc_example_expectation_row_conditions" tests/integration/test_script_runner.py
```

Comment:
This is `doc_example_` while the others are `docs_example_`. It's like this throughout so the tests do run and succeed. This doesn't need to be fixed but might be nice for consistency.